python/pyspark/ml/tests/test_image.py

```diff
@@ -67,47 +67,6 @@ def test_read_images(self):
                 lambda: ImageSchema.toImage("a"))
 
 
-class ImageFileFormatOnHiveContextTest(PySparkTestCase):
```

Comment:
@srowen, this was removed (see https://github.com/apache/spark/pull/25684#discussion_r321565529)